example-es6/index.html

```diff
@@ -0,0 +1,9 @@
+<html>
+  <head>
+    <script src="/dist/bundle.js"></script>
```

Comment:
Where does dist/bundle.js come from? I don't see it in the webpack. Seems like only worker-init.js is relevant here.